web-console/knikubevirt/snapshots/snapshots.md

```diff
@@ -14,32 +11,37 @@ When clicking on the kebab menu in the VM list, the user will be able to take a
 ![Offline Snapshot modal - default](img/1-01.jpg)
 In the basic modal, the snapshot will take all current configuration and selected parameters.
 The snapshot name is auto-generated to the current date of creation. The user can add a descriptive name.
+
 ![Offline Snapshot modal - selecting disks](img/1-02.jpg)
```

Comment:
assuming this would also cover the CD-ROM disks would we distinguish between them?

Reply:
We do distinguish between Disks and CD-ROMs on the VM-boot-order. So yes, we should do the same here.
https://github.com/Ranelim/openshift-origin-design/blob/vm-boot-order/web-console/knikubevirt/vm-details/vm-boot-order/vm-boot-order.md

Are there any other CD-ROM/disks lists that we should take into account? Does the Create-VM-Storage step include some kind of CD-ROM display?

If not, then I'll update the design to show a CD-ROM instance.

Reply:
Currently no, the CD-ROM shows currently only under advanced > Virtual hardware.